corehq/tabs/templatetags/menu_tags.py

```diff
@@ -97,15 +99,29 @@ def render(self, context):
         except DomainMembershipError:
             role_version = None
 
+        lang = get_language()
+
         from corehq.apps.hqwebapp.utils.bootstrap import get_bootstrap_version, BOOTSTRAP_5
         bootstrap_version = get_bootstrap_version()
+        use_bootstrap5 = get_bootstrap_version == BOOTSTRAP_5
```

Comment:
shouldn't `get_bootstrap_version` be `bootstrap_version in this line? or call the method?

Reply:
also, you can test a bootstrap 5 enabled view by going to
`/a/{domain}/settings/project/internal/bootstrap5/`

Reply:
Will do, thanks

Reply:
Made the requested changes and deployed to staging. Ran a creating, deleting, and restoring an application, and verified the bootstrap3 and 5 pages to verify that the applications dropdown was updated as we'd expect.